core/cmd/remote_client.go

```diff
@@ -1192,6 +1193,67 @@ func normalizePassword(password string) string {
 	return url.PathEscape(strings.TrimSpace(password))
 }
 
+// SetLogLevel sets the log level on the node
+func (cli *Client) SetLogLevel(c *clipkg.Context) (err error) {
+	if !c.Bool("level") {
+		return cli.errorOut(errors.New("expecting a log level (debug, info, warn, error)"))
+	}
+
+	logLevel := c.Args().Get(0)
+	request := web.LogPatchRequest{Level: logLevel}
+	requestData, err := json.Marshal(request)
+	if err != nil {
+		return cli.errorOut(err)
+	}
+
+	buf := bytes.NewBuffer(requestData)
+	resp, err := cli.HTTP.Patch("/v2/log", buf)
+	if err != nil {
+		return cli.errorOut(errors.Wrap(err, "from toggling debug logging"))
```

Comment:
toggling no longer makes sense - I think you can just `cli.errorOut(err)` directly here